--- Makefile.orig	2015-08-14 19:29:26.000000000 -0300
+++ Makefile	2015-08-14 21:32:39.000000000 -0300
@@ -30,8 +30,8 @@
 # number of ways. See the man page and the documentation in
 # doc/graft.{html,pdf,ps,txt}.
 
-PACKAGEDIR	= /pkgs
-TARGETDIR	= /pkgs
+PACKAGEDIR	= /usr/packages
+TARGETDIR	= /usr
 
 # You should only need to change the value of TOP to reflect the
 # installation directory for graft if it is different from your default
