From 3cdb942c6b3e45dfdcc21f521babf7ba318d0566 Mon Sep 17 00:00:00 2001
From: rofl0r <retnyg@gmx.net>
Date: Mon, 12 May 2014 18:54:00 +0200
Subject: [PATCH] libintl.h: using parens for gettext_noop breaks grep

---
 include/libintl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/libintl.h b/include/libintl.h
index 15fb5bf..b7123a9 100644
--- a/include/libintl.h
+++ b/include/libintl.h
@@ -13,7 +13,7 @@ char *bind_textdomain_codeset(const char *domainname, const char *codeset);
 char *bindtextdomain(const char *domainname, const char *dirname);
 
 #undef gettext_noop
-#define gettext_noop(X) (X)
+#define gettext_noop(X) X
 
 #ifndef LIBINTL_NO_MACROS
 /* if these macros are defined, configure checks will detect libintl as
-- 
1.8.4

